### ISSUE TITLE
How to send via ajax labels

### ISSUE THREAD

**Issue by Plasto13** [ConsoleTVs/Charts#479]
My controller for view
```
public function detail(Item $item)
    {
        $chart = new ConsumptionItem;
        $chart->label('Consumption');
        $chart->load(route('api.guest.consumption.chart.item', $item->id));

        return view('consumption::admin.items.detail', compact('item', 'chart'));
    }
```

api controller
```
public function itemChart(Item $item, Request $request)
    {
    	$data = $item->records()->select(
            DB::raw('sum(quantity) as sums'), 
            DB::raw("DATE_FORMAT(created_at,'%d.%m.%Y') as day")
  			)->groupBy('day')->get();
    	$chart = new ConsumptionItem;
    	$data = $data->pluck('sums', 'day')->toArray();
    	$keys = array_keys($data);
        $chart->labels(array_keys($data));

	$chart->dataset($item->title, 'line', array_values($data));
        return $chart->api();
    }
```
Library Highcharts
He does not show Labels, do you know someone help me??
Sry for me english

**Comment by ConsoleTVs** [comment]
The labels are not passed via ajax. This is how it works at least for now... Just data is passed. You'll need to control it in another way.
